config_defaults_inc.php

```diff
@@ -4729,3 +4729,9 @@
  */
 $g_webservice_version_when_not_found = '';
 
+/**
+ * Enables or disables @ mentions feature.
+ *
+ * @global integer $g_mentions_enabled
+ */
+$g_mentions_enabled = ON;
```

Comment:
- this config does not belong in webservice section, suggest to move it to *MantisBT Misc Settings*, maybe next to the related *bug_link_tag*
- maybe call this `g_user_mentions_enabled` to be more accurate